samples/snippets/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java

```diff
@@ -22,33 +22,58 @@
 import com.google.cloud.ServiceOptions;
 import com.google.cloud.asset.v1.AssetServiceClient;
 import com.google.cloud.asset.v1.BigQueryDestination;
+import com.google.cloud.asset.v1.ContentType;
 import com.google.cloud.asset.v1.ExportAssetsRequest;
 import com.google.cloud.asset.v1.ExportAssetsResponse;
 import com.google.cloud.asset.v1.OutputConfig;
+import com.google.cloud.asset.v1.PartitionSpec;
 import com.google.cloud.asset.v1.ProjectName;
 
 public class ExportAssetsBigqueryExample {
 
   // Use the default project Id.
   private static final String projectId = ServiceOptions.getDefaultProjectId();
 
-  // Export assets for a project.
-  // @param args path where the results will be exported to.
-  public static void exportBigQuery(String bigqueryDataset, String bigqueryTable) throws Exception {
+  // Export assets to BigQuery for a project.
+  public static void exportBigQuery(
+      String bigqueryDataset,
+      String bigqueryTable,
+      ContentType contentType,
+      boolean isPerType) throws Exception {
```

Comment:
Still need a more precise exception here.

Reply:
Add in the last commit